### PR TITLE
Feature/add name to stream

### PR DESCRIPTION
**Pull request by tompesman**
Hi Jeffery,

I've added to name to the Stream module. So it can be referenced from the consumer. I've also added some documentation on the usage.

Cheers!

### CODE REVIEW

**Review by jeffutter — COMMENTED** [review]
@tompesman First off. Thanks for the PR's. I'm thrilled someone else is using this!

File: README.md [hunk 1]
```diff
@@ -38,3 +38,50 @@ use Mix.Config
...
+  end
+
+  def handle_events(events, _from, state) do
+    IO.inspect events
+    {:noreply, [], state}
+  end
+end
+```
+
+Start the workers in de `Application`.
```
Comment (jeffutter) [inline README.md]: "the `Application`"

File: lib/particle/stream.ex [hunk 2]
```diff
@@ -20,8 +20,8 @@ defmodule Particle.Stream do
   defdelegate stream(stages), to: GenStage
   defdelegate stop(stage, reason \\ :normal, timeout \\ :infinity), to: GenStage
 
-  def start_link(url, options \\ []) do
-    GenStage.start_link(__MODULE__, url, options)
+  def start_link(url) do
+    GenStage.start_link(__MODULE__, url, name: __MODULE__)
```
Comment (jeffutter) [inline lib/particle/stream.ex]: Won't this make it impossible to start more than one stream at a time? Before, when passing options, you could  specify your own name, thus having multiple instances.

**Review by jeffutter — CHANGES_REQUESTED** [review]

File: lib/particle/stream.ex (hunk 2 above)
Comment (tompesman) [reply lib/particle/stream.ex]: You're right! I'll remove this.

File: README.md (hunk 1 above)
Comment (tompesman) [reply README.md]: 😬

**Review by jeffutter — APPROVED** [review]

**Comment by jeffutter** [comment]
Nice! Thanks for the contributions. Would you like me to cut a new version on hex.pm?

**Comment by tompesman** [comment]
That would be nice! 👍 Thank you!

**Comment by jeffutter** [comment]
@tompesman v0.1.6 pushed

**Pull request merged**